translations/monero-core_de.ts

```diff
@@ -998,17 +998,17 @@
     <message>
         <location filename="../pages/Settings.qml" line="267"/>
         <source>Password</source>
-        <translation type="unfinished">Passwort</translation>
+        <translation>Passwort</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="302"/>
         <source>Layout settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Layout-Einstellungen</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="319"/>
         <source>Custom decorations</source>
-        <translation type="unfinished"></translation>
+        <translation>Angepasste Oberfläche</translation>
```

Comment:
At the moment it doesn't really matter, so we gotta see how they plan to use it before we can really say anything. Guess we leave it like this then.